packages/core/core/src/requests/AssetGraphRequest.js

```diff
@@ -78,8 +78,14 @@ export default function createAssetGraphRequest(
     id: input.name,
     run: async input => {
       let prevResult = await input.api.getPreviousResult<AssetGraphRequestResult>();
+      let previousAssetGraphHash = prevResult?.assetGraph.getHash();
```

Comment:
The asset graph hash is needed for getting the associated bundle graph; therefore, pass the previous asset graph has back so the bundle graph request can check if a bundle graph already existed.